packages/core/helpers/coveUpdateWorker.ts

```diff
@@ -34,8 +34,4 @@ export const coveUpdateWorker = config => {
   return genConfig
 }
 
-const asyncWorker = async config => {
-  return await coveUpdateWorker(config)
-}
-
-export default asyncWorker
+export default coveUpdateWorker
```

Comment:
@adamdoe @mpallansch it's confusing now that coveUpdateWorker is both the default and a regular export from this file.